winml/lib/Api.Image/CpuTensorizer.h

```diff
@@ -93,6 +107,11 @@ class CpuTensorizer {
     else {
       return E_INVALIDARG;
     }
+
+    if (pixelRange != ImageNominalPixelRange::kNominalRange_0_255) {
+      NormalizeData(pCPUTensor, totalElements, scale, shift);
+    }
```

Comment:
It would be better to normalize during the existing double loops and take advantage of the cache rather than loop through the whole image again